### PR TITLE
ran the code beautifier, which now removes accented letters and cleans text and LaTeX files

### PR DESCRIPTION
**Pull request by komatits**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by komatits** [comment]
It seems BuildBot is off (no answer after 15 minutes), thus accepted manually.

File: utils/DSM_FOR_SPECFEM3D/OLD--VM_mesher_now_handled_by_meshfem3d/mesh_chunk.f90 [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ end subroutine compute_rotation_matrix
 ! Vadim Monteiller Mars 2013
 !
 !-------------------------------------------------------------------------------
-! matrice de rotation 3D d'axe "axe" et d'angle theta (d°)
+! matrice de rotation 3D d'axe "axe" et d'angle theta (d?)
```
Comment (QuLogic) [inline utils/DSM_FOR_SPECFEM3D/OLD--VM_mesher_now_handled_by_meshfem3d/mesh_chunk.f90]: Should write out degrees here.

File: utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/cart2geogr.f90 [hunk 2]
```diff
@@ -100,7 +100,7 @@ subroutine  compute_rotation_matrix(rotation_matrix,lon_center_chunk,lat_center_
   end subroutine compute_rotation_matrix
 
 !--------------------------------------------------------------------------------------------------------------------------------------------------------
-! 3D rotation matrix with axis  "axe" and angle theta (d°)
+! 3D rotation matrix with axis  "axe" and angle theta (d?)
```
Comment (QuLogic) [inline utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/cart2geogr.f90]: And here.

File: utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/geogr2cart.f90 [hunk 3]
```diff
@@ -102,7 +102,7 @@ subroutine  compute_rotation_matrix(rotation_matrix,lon_center_chunk,lat_center_
   end subroutine compute_rotation_matrix
 
 !--------------------------------------------------------------------------------------------------------------------------------------------------------
-! 3D rotation matrix with axis  "axe" and angle theta (d°)
+! 3D rotation matrix with axis  "axe" and angle theta (d?)
```
Comment (QuLogic) [inline utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/geogr2cart.f90]: And here.

Comment (komatits) [inline utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/geogr2cart.f90]: Thanks! I fixed all of them.

On 01/22/2015 04:56 AM, Elliott Sales de Andrade wrote:
> In
> utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/geogr2cart.f90 <https://github.com/geodynamics/specfem3d/pull/366#discussion_r23354308>:
>
>> @@ -102,7 +102,7 @@ subroutine  compute_rotation_matrix(rotation_matrix,lon_center_chunk,lat_center_
>>    end subroutine compute_rotation_matrix
>>
>>  !--------------------------------------------------------------------------------------------------------------------------------------------------------
>> -! 3D rotation matrix with axis  "axe" and angle theta (d°)
>> +! 3D rotation matrix with axis  "axe" and angle theta (d?)
>
> And here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d/pull/366/files#r23354308>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

**Pull request merged**